src/SDKs/Resource/Management.ResourceManager/Microsoft.Azure.Management.ResourceManager.csproj

```diff
@@ -7,7 +7,7 @@
 		<PackageId>Microsoft.Azure.Management.ResourceManager</PackageId>
 		<Description>Provides resource group and resource management capabilities for Microsoft Azure.</Description>
 		<AssemblyName>Microsoft.Azure.Management.ResourceManager</AssemblyName>
-		<Version>1.7.0-preview</Version>
+		<Version>1.8.0-preview</Version>
 		<PackageTags>Microsoft Azure resource management;resource management;resource groups;</PackageTags>
```

Comment:
Please add `PackageReleaseNotes` similar to [this](https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/Compute/Management.Compute/Microsoft.Azure.Management.Compute.csproj#L13)

Reply:
@dsgouda I run msbuild build.proj /t:Build /p:Scope=SDKs\Resource but didn't see any change being generated. Did I understand it correctly?
Can you give more details about generate.ps1? today we have generate.cmd: https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/Resource/Management.ResourceManager/generate.cmd, is that  not the right way?

Reply:
@Tiano2017 
If msbuild did not generate any artifacts, the PR should be fine
We are moving away from using .cmd script to generate the code. Hence the .ps1 script.

Reply:
'Start-AutoRestCodeGeneration' is not recognized. Do I need to install anything?

Reply:
@dsgouda any clue on this?

Reply:
Please pull the latest upstream changes and run `msbuild build.proj` this should install the cmdlet for you